extern console

import '../export/export.throw.extern.ne'

try {
	foo()
}
catch error {
	console.error(error)
}